### PR TITLE
Rmd, csv+xlsx data files for groundwater data example

### PR DESCRIPTION
**Pull request by aappling-usgs**
@wdwatkins - csvs and xlsx with the same groundwater data: 5 of the most-measured wells in Nevada. Also including the script I used to create the data files (create_data.R) and a sketch of activities to do with these data during the workshop (gw_analysis.Rmd). If we follow that sketch, users will need to have these libraries:

- data.table
- XLConnect
- dplyr
- ggplot2

(in addition to dataRetrieval and any others already on your list)

Feedback/edits welcome. I'll be pretty busy until Wednesday afternoon. If you think this is good enough to run with, please send out the data files with your email tomorrow (Monday).

### CODE REVIEW

**Comment by lindsaycarr** [comment]
@aappling-usgs I would just caution against using `XLConnect` because it requires that users have Java. Instead, I would look into `readxl`. Unfortunately `readxl` will not allow you to write Excel files, but I'd say keep it for the `create_data.R` file and add a note that the package requires Java. I would just avoid using it during the actual course workflow for students.

**Comment by lindsaycarr** [comment]
Also, we can only have one .Rmd per course with the way the site is setup. So the example can be a new section in the existing `content/course-specific-material/introR-NatGW-Aug16.Rmd` file. Additionally, you would need to link to the create_data.R or csv files in this .Rmd for the students to get to them. I'd be happy to move things around after this PR since you're pretty busy

**Comment by ldecicco-USGS** [comment]
Not only Java...but a specific, changing, version of Java.

**Comment by aappling-usgs** [comment]
Thanks for the readxl/XLConnect thoughts. readxl is fine with me.

for the Rmd file, I sort of envisioned keeping the one I PRed in our back pocket, typing it out for students during the workshop, and copying that content into `content/course-specific-material/introR-NatGW-Aug16.Rmd` after the workshop. what do you think?

**Comment by wdwatkins** [comment]
That sounds fine for the .Rmd.  

My only other comment would be do we want any NAs in the levels csv?  I only saw them in the site info file.

**Comment by aappling-usgs** [comment]
There are NAs in the levels csv. They're just buried deep down somewhere.
```
> gwlevels <- fread(input = 'gwlevels.csv', data.table=FALSE)
> which(is.na(gwlevels$level_ft))
 [1] 3313 3314 3315 3316 3317 3318 3319 3320 3321 3960 3961 3963 3964 3966 3967 3968 3969 3970 3971 3972 3973
[22] 3974 3975 3976
```

**Comment by wdwatkins** [comment]
Ah ok didn't see them.

File: content/course-specific-material/introR-NatGW-Aug29/gw_analysis.Rmd [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+```
+
+Load data from .xlsx file. See also http://www.milanor.net/blog/steps-connect-r-excel-xlconnect/
+
+```{r}
+# load the data and metadata
+library(XLConnect)
+workbook <- loadWorkbook('gwlevels.xlsx')
+gwlevels <- readWorksheet(workbook, 'gwlevels')
+gwsiteinfo <- readWorksheet(workbook, 'gwsiteinfo', startRow = 3, startCol = 2)
```
Comment (lindsaycarr) [inline content/course-specific-material/introR-NatGW-Aug29/gw_analysis.Rmd]: @aappling-usgs what is the purpose of this?

**Comment by aappling-usgs** [comment]
i wrote the worksheet such that the data doesn't actually start until row 3, col 2.  we could make the xlsx cleaner, but i thought it was a useful illustration of a common formatting issue with Excel files

**Comment by lindsaycarr** [comment]
Ok, I see that now. It seems that readxl has automatically decided to skip over those rows and columns

**Comment by aappling-usgs** [comment]
way to go, readxl! very cool

**Pull request merged**